vcell-core/src/main/java/cbit/util/kisao/KisaoTermParser.java

```diff
@@ -22,16 +24,15 @@ public class KisaoTermParser {
 	final Pattern ISA_PATTERN = Pattern.compile("is_a:\\s*(\\S+)");
 	
 	KisaoOntology parse() {
-		InputStream is = KisaoTermParser.class.getClassLoader().getResourceAsStream(Kisao_OBO);
-		BufferedReader isr = new BufferedReader(new InputStreamReader(is));
-		String line = null;
-		boolean inState = false;
 		KisaoOntology ontology = new KisaoOntology();
 		KisaoTerm curr = null;
+		boolean inState = false;
+
+		InputStream is = KisaoTermParser.class.getClassLoader().getResourceAsStream(Kisao_OBO);
+        assert is != null;

```

Comment:
assert's are not great in production code - just throw some sort of IOException or Runtime (assert's are not designed to be handled in production).